packages/compass/src/main/window-manager.js

```diff
@@ -319,6 +319,27 @@ function showAboutDialog() {
   });
 }
 
+function showLogFileDialog({ logFilePath }) {
+  dialog.showMessageBox({
+    type: 'info',
+    title: 'Log file for this session',
+    icon: COMPASS_ICON,
+    message: `The log file for this session can be found at ${logFilePath}`,
```

Comment:
It wasn't able to open one of the log files until after Compass was exited:
<img width="384" alt="Screen Shot 2021-09-24 at 1 56 52 PM" src="https://user-images.githubusercontent.com/1791149/134720047-37072c74-2676-4b94-90ee-853df5145893.png">
Do we want to include something about that behavior in this message? Is this expected?

Reply:
> Do we want to include something about that behavior in this message?

Yeah … something in the detail text that says something like “Some tools may not be able to read the file until Compass has exited”? It turned out that this also fails with e.g. gunzip on macOS, but not gunzip on Linux.

> Is this expected?

Yeah, Compass doesn’t fully close the file until it exits, so it will always look like an incomplete gzip stream while it’s still running. I guess we could also provide a way to get the log file contents as text from the dialog here, but I’m not sure if that’s not a bit much effort for relatively little gain?

Reply:
Also … I guess if we wanted to, we could read the incomplete file and decompress it ourselves here, and then just provide it to the user as a plain text file. How does that sound? Would that be something we would want to expose instead of the current approach here?

Reply:
> Some tools may not be able to read the log file until Compass has exited
sounds good yup

> I guess we could also provide a way to get the log file contents as text from the dialog here, but I’m not sure if that’s not a bit much effort for relatively little gain?
Makes sense to me to not to support that unless we need to I think, especially since the main use case of this is to aid in debugging.